### PR TITLE
Bug fixes related to functional tests

### PR DESCRIPTION
**Pull request by jvarlamova**
While implementing functional tests there were found several small issues in Kostyor.

### CODE REVIEW

**Comment by sc68cal** [comment]
I think instead of UNDEFINED we should say - "Unknown" for versions that we cannot determine

File: kostyor/rest_api.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def discover_cluster():
         for host, service in six.iteritems(host_service_map):
             for s in service:
                 db_api.create_service(s, hosts_ids[host],
-                                      constants.NOT_READY_FOR_UPGRADE)
+                                      constants.UNDEFINED)
```
Comment (sc68cal) [inline kostyor/rest_api.py]: No I think it's actually worse to pass in undefined.

File: kostyor/db/models.py [hunk 2]
```diff
@@ -21,7 +21,7 @@ class Cluster(Base, HasId):
     __tablename__ = 'clusters'
 
     name = sa.Column(sa.String(255))
-    version = sa.Column(sa.Enum(*constants.OPENSTACK_VERSIONS))
+    version = sa.Column(sa.Enum(*constants.CLUSTER_VERSIONS))
```
Comment (sc68cal) [inline kostyor/db/models.py]: No, I wouldn't use CLUSTER_VERSIONS.

File: kostyor/rest_api.py (hunk 1 above)
Comment (jvarlamova) [inline kostyor/rest_api.py]: But method create_service expects service version, which is undefined(or unknown) - https://github.com/sc68cal/Kostyor/blob/master/kostyor/db/api.py#L101

Comment (sc68cal) [inline kostyor/rest_api.py]: My issue is, I don't like the "undefined" value. - see https://github.com/sc68cal/Kostyor/pull/24#issuecomment-241740952

Comment (jvarlamova) [inline kostyor/rest_api.py]: Ok, now I'm getting it:)

**Comment by sc68cal** [comment]
👍

**Pull request merged**